galaxy/isa_get_data_files.xml

```diff
@@ -1,9 +1,8 @@
 <?xml version='1.0' encoding='UTF-8'?>
-<tool id="data_files" name="data_files" version="1.0">
+<tool id="isa_get_data_files" name="Get data files" version="1.0">
```

Comment:
Please make sure to bump wrapper versions, specially when changing formats of tools outputs.